### ISSUE TITLE
Status?

### ISSUE THREAD

**Issue by arderyp** [miki725/django-sqrl#7]
Is this still something you are actively developing and bringing up to speed with the official Version 1 release of SQRL that was just published?

**Comment by miki725** [comment]
yes the idea to catch up with all the progress since initial sqrl discussions

**Comment by bschollnick** [comment]
I'd suggest, starting with a more up to date implementation.  I'm unable to contact the fork owner, but working from code that has already been updated to python 3, and a later version of django, might be a worthwhile endeavor.

**Also, please add this to PyPi.**  That's vital for anyone to use it, since it allows easy upgrading of the library.  (It's not hard  
https://the-hitchhikers-guide-to-packaging.readthedocs.io/en/latest/creation.html

https://git.kiichan.space/kiichan/django-sqrl-2
